### ISSUE TITLE
rotation:degrees= ?

### ISSUE THREAD

**Issue by conzorz** [Clans/FloatingActionButton#269]
Is there an option to specify how much you want the FAB to rotate when pressed?

**Comment by Clans** [comment]
No there is not.

**Comment by Ahsan221B** [comment]
Yes there is! you can create an animation in anim folder and then apply it on                     app:menu_labels_showAnimation="" or you can use android:rotation property programmatically.
